localstack/services/cloudwatch/alarm_scheduler.py

```diff
@@ -47,7 +47,7 @@ def shutdown_scheduler(self) -> None:
         Shutsdown the scheduler, must be called before application stops
         """
         self.scheduler.close()
-        self.thread.join(5)
+        self.thread.join(10)
```

Comment:
When I ran the tests locally, I sometimes ran into issues, where some threads did not properly shutdown. [`thread.join(x)`](https://docs.python.org/2/library/threading.html#threading.Thread.join) should wait up to x seconds to wait until the thread is closed. It will not always wait for 10 sec. I did not experience issues with the local execution when increasing the value. But I am not sure if this is just an issue when running tests in host mode.